container-images/mds-compliance/package.json

```diff
@@ -10,7 +10,6 @@
   "scripts": {
     "bundle": "webpack --mode=production --env.npm_package_name=${npm_package_name} --env.npm_package_version=${npm_package_version}",
     "image": "yarn bundle && ../../bin/mdsctl -c image-name=mds-compliance:${npm_package_version} buildImage",
-    "start": "yarn watch server",
```

Comment:
Re. my earlier comment, it looks like we are getting rid of `yarn start` in a few places. I'd like to chat about that more and figure out what the new equivalent is, if any.

Reply:
`yarn start` was only removed from `container-images/*` not `packages/*`